shared/rakefiles/entrypoint.rake

```diff
@@ -83,6 +88,28 @@ end
 desc "Create cluster and deploy GPII components to it"
 task :deploy => [:set_vars, :apply_infra] do
   sh "#{@exekube_cmd} rake xk[up]"
+  Rake::Task["display_cluster_info"].invoke
+end
+
+desc "Display some handy info about the cluster"
+task :display_cluster_info do
+  puts
+  puts
+  puts "*************************************************"
+  puts "Congratulations! Your GPII Cloud in GCP is ready!"
+  puts "*************************************************"
+  puts
+  puts "GCP Dashboard:"
+  puts "  https://console.cloud.google.com/home/dashboard?organizationId=#{ ENV["TF_VAR_organization_id"] }&project=#{ ENV["TF_VAR_project_id"] }"
+  puts
+  puts "StackDriver log console:"
```

Comment:
It's "Stackdriver" :)

Reply:
I would name this `Stackdriver Logging Dashboard:`, and would also add `Stackdriver Monitoring Dashboard:` with `https://app.google.stackdriver.com/?project=#{ENV["TF_VAR_project_id"]}`.